legend-engine-xt-nonrelationalStore-mongodb-grammar/src/main/java/org/finos/legend/engine/language/pure/grammar/from/MongoDBParserException.java

```diff
@@ -12,16 +12,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package org.finos.legend.engine.language.mongodb.schema.grammar.from;
+package org.finos.legend.engine.language.pure.grammar.from;
```

Comment:
why we move it from the mongo package?

Reply:
was following the pattern here - most of the other modules keep the grammar under language.pure (so did this change). not sure what the right one is.